chainercv/links/detection_link.py

```diff
@@ -0,0 +1,39 @@
+import chainer
+
+
+class DetectionLink(chainer.Link):
```

Comment:
Actually, I don't like multiple inheritance, either. However, I don't like to assume that all detectors inherit `chainer.Chain` although there are no examples inheriting `chainer.ChainList`.
One simple solution is to remove `DetectionLink` and define APIs as a document. We can choose base class for each detector without multiple inheritance.
(cf. `ResNet` and `VGG16` have similar APIs, but they don't share any base classes.)